src/rules/onfido-idv.js

```diff
@@ -0,0 +1,108 @@
+/* global configuration */
+
+/**
+ * @title Onfido Identity Verification
+ * @overview Redirect to your Onfido IDV Application for Identity Verification during login.
+ * @gallery true
+ * @category marketplace
+ * 
+ * Onfido digitally prove people’s real identities using a photo ID and facial biometrics. So your users can verify themselves anywhere, anytime. 
+ * By leveraging Onfido with Auth0, we enable organizations to know who their digital user are, tied to a real identity, during authentication, onboarding, other high-risk actions.
+ * This rule will redirect to a custom application you build implementing one of the Onfido Input Capture SDKs. More info: https://developers.onfido.com/
+ * 
+ * Use Rules configuration to define:
+ *    - SESSION_TOKEN_SECRET: Long, random string, should match on Onfido app side.
+ *    - ONFIDO_ID_VERIFICATION_URL: URL to receive the redirect
+ *    - ONFIDO_API_TOKEN: Your Onfido API Token
+ *    - ONFIDO_REGION: The supported Onfido region your tenant is operating in
+ *
+ * @param {object} user
+ * @param {object} context
+ * @param {function} callback
+ */
+async function redirectOnfidoRule(user, context, callback) {
```

Comment:
```suggestion
async function onfidoIdentityVerification(user, context, callback) {
```
This matches the Rule name to connect things a little better.

Reply:
agreed. Oversight on my part.